src/Athena.Core/Models/Campus.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Athena.Core.Models
+{
+    public class Campus : IUniqueObject<Guid>
+    {
+        /// <inheritdoc />
+        public Guid Id { get; set; }
+
+        /// <summary>
+        /// The name of the campus
+        /// </summary>
+        public string Name { get; set; }
+        
+        /// <summary>
+        /// A description of the campus
+        /// </summary>
+        public string Description { get; set; }
+        
+        /// <summary>
+        /// The address of the campus
+        /// </summary>
+        public string Location { get; set; }
+
+        /// <summary>
+        /// The institutions that offer courses on this campus
+        /// </summary>
+        public List<Institution> Institutions { get; } = new List<Institution>();
```

Comment:
I feel this could be fetched in a separate action, I don't think we need to return it with the campus